### PR TITLE
added functionality to post comment

### PR DESCRIPTION
**Pull request by nldblanch**
I've made it so that it renders the comment immediately in a "fake comment card" while it sends the comment to the database. When it is offline, it can't reach the database and it crashes. I am unsure right now how to address that

### CODE REVIEW

File: src/components/PostComment.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      created_at: new Date(),
+    });
+    setIsLoading(true);
+    postComment(article_id, { username: "grumpy19", body: comment })
+      .then((comment) => {
+        setComments((prev) => [comment, ...prev]);
+        event.target[0].value = "";
+        setIsLoading(false);
+      })
+  };
```
Comment (nc-rayhaan) [inline src/components/PostComment.jsx:26]: This isn't working for me. When I try to post a comment, it refreshes but the new comment isn't there. As it stands, the form is not a controlled component. Ideally, you want to save whatever the user types in state and then use the state to pass to the new comment object.

File: src/components/PostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <>
+      <form id="comment-box" onSubmit={handleSubmit}>
+        <input placeholder="comment" type="text" />
+        <button type="submit">Post</button>
+      </form>
+      {isLoading && <FakeCommentCard comment={fakeComment} />}
```
Comment (nc-rayhaan) [inline src/components/PostComment.jsx:34]: Try and see if you can get this working without the faceCommentCard

**Review by nc-rayhaan — COMMENTED** [review]
Not quite there yet but not far off! Please do fix and then resubmit

File: src/components/PostComment.jsx (hunk 2 above)
Comment (nldblanch) [reply src/components/PostComment.jsx:34]: I figured out why - I took out Error / setError but left a function call in there, so it was getting caught. The fake coment card is a REALLY nice visual that makes sense when you see it

Comment (nc-rayhaan) [reply src/components/PostComment.jsx:34]: Ok, I just tried the working version and yes it looks great! Ok I'm on board! 😄

**Pull request merged**